backend/modules/sse/__init__.py

```
@@ -0,0 +1,74 @@
+import uuid
+from typing import override
```

Pylint (3.3.7)

[error] No name 'override' in module 'typing'

(E0611)